import-automation/progress-dashboard-rest/app/service/base_database.py

```diff
@@ -106,14 +106,18 @@ def filter(self, kv_dict):
 
         Args:
             kv_dict: Key-value mappings used for filtering as a dict.
+            order: Sequence of field names each as a string to order
```

Comment:
I think you are specifying a tuple, not a sequence.

Reply:
fixed